interactors/src/test/java/com/thefuntasty/interactors/disposables/SingleDisposablesOwnerTest.kt

```diff
@@ -114,4 +114,25 @@ class SingleDisposablesOwnerTest : RxMockitoJUnitRunner() {
         assertTrue(!disposablesList[0].isDisposed)
         assertTrue(!disposablesList[1].isDisposed)
     }
-}
\ No newline at end of file
+
+    @Test
+    fun `run execute without params`() {
+        val tempSingler = object : BaseSingler<Unit, String>() {
+            override fun prepare(args: Unit) = Single.just("Hello")
+        }
+
+        withDisposablesOwner {
+            tempSingler.execute {
+                onSuccess {
+                    assertEquals(it, "Hello")
```

Comment:
This is not a good test since you can't be sure that this line was executed. You should capture string to some local variable and assert at the end of the test that it is working correctly.